src/components/calcite-radio-group-item/calcite-radio-group-item.scss

```diff
@@ -1,101 +1,108 @@
 :host {
-  display: flex;
-  align-self: stretch;
-  cursor: pointer;
-  transition: background-color 0.1s ease-in-out, border-color 0.1s ease-in-out;
```

Comment:
Might want to retain the original transition here. `transition-color` means we're applying transition effect to `color` property, not `background-color` and `border-color` as it was before